{"uuid":"4a7cce0c-c85d-4a29-b002-5dc218c06383","name":"An uncaught error was detected outside of a test","historyId":"102bf66623d6956690ecad0db5edf6a5:d41d8cd98f00b204e9800998ecf8427e","status":"broken","statusDetails":{"message":"The following error originated from your test code, not from Cypress.\n\n  > BeforeEach is not defined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","trace":"ReferenceError: The following error originated from your test code, not from Cypress.\n\n  > BeforeEach is not defined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at Suite.eval (webpack://desafio-api-test/./cypress/e2e/api/produtos/delete_produto.cy.js:4:4)\n    at Object.create (https://serverest.dev/__cypress/runner/cypress_runner.js:164545:19)\n    at context.describe.context.context (https://serverest.dev/__cypress/runner/cypress_runner.js:164232:27)\n    at createRunnable (https://serverest.dev/__cypress/runner/cypress_runner.js:148247:31)\n    at <unknown> (https://serverest.dev/__cypress/runner/cypress_runner.js:148308:14)\n    at Xr (webpack://desafio-api-test/./node_modules/allure-cypress/dist/cjs/index.js:1:19441)\n    at eval (webpack://desafio-api-test/./node_modules/allure-cypress/dist/cjs/index.js:1:19175)\n    at eval (webpack://desafio-api-test/./cypress/e2e/api/produtos/delete_produto.cy.js:1:0)\n    at eval (https://serverest.dev/__cypress/tests?p=cypress\\e2e\\api\\produtos\\delete_produto.cy.js:90:12)\n    at eval (<anonymous>)\nFrom previous event:\n    at Promise.longStackTracesCaptureStackTrace [as _captureStackTrace] (https://serverest.dev/__cypress/runner/cypress_runner.js:3456:19)\n    at Promise._then (https://serverest.dev/__cypress/runner/cypress_runner.js:1209:17)\n    at Promise.then (https://serverest.dev/__cypress/runner/cypress_runner.js:1102:17)\n    at runScriptsFromUrls (https://serverest.dev/__cypress/runner/cypress_runner.js:112906:136)\n    at Object.runScripts (https://serverest.dev/__cypress/runner/cypress_runner.js:112947:12)\n    at $Cypress.onSpecWindow (https://serverest.dev/__cypress/runner/cypress_runner.js:41405:67)\n    at <unknown> (https://serverest.dev/__cypress/iframes/cypress%5Ce2e%5Capi%5Cprodutos%5Cdelete_produto.cy.js?browserFamily=chromium:276:22)"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"labels":[{"name":"language","value":"javascript"},{"name":"framework","value":"cypress"},{"name":"host","value":"LAPTOP-BUJ0A66L"},{"name":"thread","value":"pid-24924-worker-main"},{"name":"package","value":"cypress.e2e.api.produtos.delete_produto.cy.js"}],"links":[],"start":1748610411874,"fullName":"cypress/e2e/api/produtos/delete_produto.cy.js#An uncaught error was detected outside of a test","testCaseId":"102bf66623d6956690ecad0db5edf6a5","stop":1748610411914}